### ISSUE TITLE
smartylist tarbomb

### ISSUE THREAD

**Issue by ftrotter** [smartystreets/smartystreets-php-sdk#3]
Seriously? a tarbomb in 2018? Poor form. 

http://www.linfo.org/tarbomb.html

-FT

**Comment by cbuchert** [comment]
Where are you seeing this?

**Comment by SunSparc** [comment]
@ftrotter, I am not sure what you are referring to. Some more details would be helpful.

I was able to go to our [releases page](https://github.com/smartystreets/smartystreets-php-sdk/releases), click on the `tar.gz` next to the `4.2.0` release and download the `smartystreets-php-sdk-4.2.0.tar.gz` file. On the command line I ran: `tar zxvf smartystreets-php-sdk-4.2.0.tar.gz`. It created a subdirectory called, "smartystreets-php-sdk-4.2.0" with the content of the tarball.

I am on macOS.

What are you seeing? What system? Steps to duplicate?
